code/src/LikeButton.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react"
+import "./likebutton.css"
+export const LikeButton = (props) => {
+    const { hearts, id } = props
+    const [heartCounter, setHeartCounter] = useState(hearts)
+    const [youLiked, setYouLiked] = useState(+localStorage.getItem(`${id}`) || 0)
```

Comment:
Proposal of new naming 
```suggestion
    const [liked, setLiked] = useState(+localStorage.getItem(`${id}`) || 0)
```